src/components/transcript/index.tsx

```diff
@@ -216,6 +216,7 @@ const Transcript = ({ reviewData }: { reviewData: UserReviewData }) => {
         directoryPath: loc?.trim() ? loc.trim() : config.defaultDirectoryPath,
         fileName: formatDataForMetadata(title),
         url: transcriptData?.content.media,
+        prUrl: reviewData?.pr_url || undefined,
```

Comment:
why or undefined?

Reply:
i see now that's it's a repition  reviewData?.pr_url would return undefined when there's no pr_url.